devtools/ci/ci_main.sh

```
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -u
+set +e
+is_self_runner=`echo $RUNNER_LABEL | awk -F '-' '{print $1}'`
+if [[ $is_self_runner == "self" ]];then
+  CARGO_TARGET_DIR=$GITHUB_WORKSPACE/../target
+fi
+CARGO_TARGET_DIR=${CARGO_TARGET_DIR:-"$GITHUB_WORKSPACE/target"}
+case $GITHUB_WORKFLOW in
+  ci_linters*)
+    echo "Hellow ci_linters!"
+    cargo fmt --version ||  rustup component add rustfmt
+    argo clippy --version ||  rustup component add clippy
```

typo issue, please change from "argo" to "cargo",  and "||" hide the error.